cli/crates/server/src/servers.rs

```diff
@@ -123,7 +124,15 @@ async fn spawn_servers(
 
     let environment_variables: std::collections::HashMap<_, _> = crate::environment::variables().collect();
 
-    let resolvers = run_schema_parser(&environment_variables).await?;
+    let resolvers = match run_schema_parser(&environment_variables).await {
+        Ok(resolvers) => resolvers,
+        Err(error) => {
+            let _ = sender.send(ServerMessage::CompilationError(error.to_string()));
+            tokio::spawn(async move { error_server::start(worker_port, error.to_string(), bridge_event_bus).await })
```

Comment:
We could do the same for failed resolver compilation?

Reply:
Adding this as well